Corona-Warn-App/src/main/res/values/risk_details_strings.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <!-- XHED: Hygiene rules title -->
+    <string name="risk_details_hygiene_rules_title">Hygieneregeln</string>
+    <!-- XHED: Home rules title -->
+    <string name="risk_details_stay_at_home_title">Ansteckungsrisiko minimieren</string>
+    <!-- XTXT: risk details - hygiene info -->
+    <string name="risk_details_behavior_body_hygiene">Bitte achten Sie jetzt besonders auf die üblichen Hygieneregeln.</string>
+    <!-- XTXT: risk details - meetings info -->
+    <string name="risk_details_behavior_body_meetings">Treffen und Veranstaltungen: Verzichten Sie auf Besuche und Aktivitäten, bei denen Sie mit anderen Menschen zusammenkommen (Clubs, Bars, Restaurants, Kino, Theater, Sportveranstaltungen).</string>
+    <!-- XTXT: risk details - home office info -->
+    <string name="risk_details_behavior_body_home_office">Home Office: Arbeiten Sie von zu Hause, wenn es für Sie möglich ist.</string>
+    <!-- XTXT: risk details - shopping info -->
+    <string name="risk_details_behavior_body_shopping">Einkaufen: Lassen Sie sich die Lebensmittel liefern oder bitten Sie jemanden, die Einkäufe für Sie zu erledigen.</string>
+    <!-- XTXT: risk details - public transport info -->
+    <string name="risk_details_behavior_body_public_transport">Öffentliche Verkehrsmittel: Meiden Sie diese so gut es geht, vor allem zu Stoßzeiten.</string>
+    <!-- XTXT: risk details - travelling info -->
+    <string name="risk_details_behavior_body_traveling">Reisen: Verzichten Sie auf Flug- und Zugreisen.</string>
+    <!-- XTXT: risk details - health department info -->
+    <string name="risk_details_behavior_body_health_department">Bei Fragen zu Quarantäne-Maßnahmen wenden Sie sich bitte an Ihr Gesundheitsamt.</string>
+    <!-- XTXT: risk details - health department info link label -->
+    <string name="risk_details_behavior_body_health_department_label">Fragen zu Quarantäne-Maßnahmen</string>
+    <!-- XTXT: risk details - health department info link url -->
+    <string name="risk_details_behavior_body_health_department_link">https://www.coronawarn.app/en/faq/#voluntary_self_quarantine</string>
+    <!-- XTXT: risk details - home rules body -->
+    <string name="risk_details_minimize_risk_body">Falls Sie sich angesteckt haben, verringern Sie so das Risiko, andere anzustecken:</string>
+    <!-- XTXT: risk details - home rules faq -->
+    <string name="risk_details_minimize_risk_faq">Weitere Hinweise finden Sie in der FAQ zur freiwilligen Selbstquarantäne.</string>
+    <!-- XTXT: risk details - home rules faq link label -->
+    <string name="risk_details_minimize_risk_faq_label">FAQ zur freiwilligen Selbstquarantäne.</string>
+    <!-- XTXT: risk details - home rules faq link url -->
+    <string name="risk_details_minimize_risk_faq_link">https://www.coronawarn.app/en/faq/#quarantine_measures</string>
```

Comment:
Sure that this is the correct link? I'm just going by the descriptions and the names of the strings (: